packages/iov-keycontrol/src/hdpaths.ts

```diff
@@ -34,6 +34,39 @@ export class HdPaths {
     ];
   }
 
+  /**
```

Comment:
Great comment here.
Love it when I learn context in the code reviews

Reply:
:) I'm glad it helps